### DISCUSSION TITLE
Markdown completion in Visual Studio may not work

### DISCUSSION THREAD

**Discussion by bertrand-alain** [community/community#121810] [Copilot Conversations]
### Select Topic Area

Bug

### Body

While feeding lists (starting with '-') copilot completion (tab) is not triggered, instead spacing is added at the begining of the row. Maybe we should be able to have another key to trigger the completion as the tab or that it prevents in that case the identing of the row?

**Comment by ayim** [comment]
I was able to reproduce this in version 17.10.
![markdown1710](https://github.com/community/community/assets/10951869/0f2b5bee-3657-4557-a0b0-0fcece35ba86)

**Update: There is a FIX PENDING RELEASE**
See here: https://developercommunity.visualstudio.com/t/Tab-does-not-work-for-GitHub-Copilot-in-/10530824

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
aayim@microsoft.com
